bin/millau/runtime/src/lib.rs

```diff
@@ -1083,7 +1083,7 @@ impl_runtime_apis! {
 				fn prepare_parachain_heads_proof(
 					parachains: &[bp_polkadot_core::parachains::ParaId],
 					parachain_head_size: u32,
-					proof_size: bp_runtime::StorageProofSize,
```

Comment:
This rename I am leaning against. `StorageSize` can be misleading as opposed to `StorageProofSize` which describes it completely.